diff --git a/exercises/020_user_data_128.sh b/exercises/020_user_data_128.sh
index 26e6a8f..f533e91 100755
--- a/exercises/020_user_data_128.sh
+++ b/exercises/020_user_data_128.sh
@@ -23,8 +23,8 @@ tb "create_accounts id=2000 code=10 ledger=200 user_data_128=${USER_1},
 
 # Can you use this payment ID to connect the two transfers?
 PAYMENT_1=9000000001
-tb "create_transfers id=20000 debit_account_id=2000 credit_account_id=2001 amount=100 ledger=200 code=10,
-                     id=20001 debit_account_id=2002 credit_account_id=2003 amount=100 ledger=201 code=10;"
+tb "create_transfers id=20000 debit_account_id=2000 credit_account_id=2001 amount=100 ledger=200 code=10 user_data_128=${PAYMENT_1},
+                     id=20001 debit_account_id=2002 credit_account_id=2003 amount=100 ledger=201 code=10 user_data_128=${PAYMENT_1};"
 
 # (Note that if you run this script a second time, you'll need to increment the transfer `id` to avoid 
 # a conflict with previously created transfers.)
