diff --git a/exercises/022_multi_debit_credit_transfers.sh b/exercises/022_multi_debit_credit_transfers.sh
index 133d033..3f5e2ff 100755
--- a/exercises/022_multi_debit_credit_transfers.sh
+++ b/exercises/022_multi_debit_credit_transfers.sh
@@ -17,6 +17,6 @@ tb "create_accounts id=22000 code=10 ledger=220,
 
 # Now, we want these transfers to atomically debit one account and credit the other three.
 # Can you help fix this request?
-tb "create_transfers id=220000 debit_account_id=22000 credit_account_id=22001 amount=100 ledger=220 code=10 flags=???,
-                     id=220001 debit_account_id=22000 credit_account_id=22002 amount=100 ledger=220 code=10 flags=???,
-                     id=220002 debit_account_id=22000 credit_account_id=22003 amount=100 ledger=220 code=10 flags=???;"
+tb "create_transfers id=220000 debit_account_id=22000 credit_account_id=22001 amount=100 ledger=220 code=10 flags=linked,
+                     id=220001 debit_account_id=22000 credit_account_id=22002 amount=100 ledger=220 code=10 flags=linked,
+                     id=220002 debit_account_id=22000 credit_account_id=22003 amount=100 ledger=220 code=10;"
