diff --git a/exercises/026_two_phase_transfers_partial.sh b/exercises/026_two_phase_transfers_partial.sh
index f3dcc0a..555d557 100755
--- a/exercises/026_two_phase_transfers_partial.sh
+++ b/exercises/026_two_phase_transfers_partial.sh
@@ -16,7 +16,7 @@ tb "create_transfers id=26000 debit_account_id=2600 credit_account_id=2601 amoun
 # Now, we'll post them -- but we're going to specify different amounts:
 output=$(tb "create_transfers id=26003 pending_id=26000 amount=0    flags=post_pending_transfer,
                               id=26004 pending_id=26001 amount=200  flags=post_pending_transfer,
-                              id=26005 pending_id=26002 amount=200  flags=post_pending_transfer;")
+                              id=26005 pending_id=26002 amount=100  flags=post_pending_transfer;")
 
 # Uh oh! That didn't work!
 # The first transfer is actually okay -- specifying an amount of 0 means the full amount is posted.
